views/imagestream-listing.html

```diff
@@ -1,4 +1,4 @@
-<table class="listing-ct">
+<table ng-class="classNames ? '{{classNames}}' : 'listing-ct'">
```

Comment:
@rhamilto have you actually tried it? For me without the class a bunch of other styles don't apply. Is the plan to just have overrides there?